README.md

```diff
@@ -95,6 +96,28 @@ Subsystem by running the following in PowerShell as an administrator
 Restart-Service LxssManager*
 ```
 
+### Local Development Login
+
+To log in to the system once locally deployed, simply enter a username in the designated field and click the "Login" button.
+
+#### Users & user types
+
+The following table lists the available usernames and their corresponding user types, as well as their associated permissions:
```

Comment:
Including documentation of the types of users may cause confusion at this time since the scopes have not been implemented yet.  Lets remove information in this merge request regarding their types and permissions for the time being.  This can get added back in when we implement the scopes for the UI & the UX.

Reply:
Agree with this, without scopes, the user types aren't relevant yet.

Additionally, perhaps it might be useful to point to where these users exist in the code base so they see what is available?

`etc/fixtures/initial-seed/users.json`

Reply:
changes made